client/src/models/Event.ts

```diff
@@ -0,0 +1,226 @@
+import { gql } from "@apollo/client"
+import Model, { GRAPHQL_ENTITY_AVATAR_FIELDS, yupDate } from "components/Model"
+import _isEmpty from "lodash/isEmpty"
+import EVENTS_ICON from "resources/events.png"
+import Settings from "settings"
+import utils from "utils"
+import * as yup from "yup"
+
+export default class Event extends Model {
+  static resourceName = "Event"
+  static listName = "eventList"
+  static getInstanceName = "event"
+  static relatedObjectType = "events"
+
+  static displayName() {
+    return "Event"
+  }
+
+  static EVENT_TYPES = {
+    OTHER: "OTHER",
+    CONFERENCE: "CONFERENCE",
+    EXERCISE: "EXERCISE",
+    VISIT_BAN: "VISIT_BAN"
+  }
+
+  static schema = {}
+
+  static yupSchema = yup.object().shape({
+    status: yup
+      .string()
+      .required()
+      .default(() => Model.STATUS.ACTIVE),
+    type: yup.string().required().default(""),
+    name: yup.string().required().default(""),
+    description: yup.string().default(""),
+    startDate: yupDate.required().default(null),
+    endDate: yupDate.required().default(null),
+    outcomes: yup.string().default(""),
+    hostOrg: yup
+      .object()
+      .test("hostOrg", "host org error", (hostOrg, testContext) =>
+        _isEmpty(hostOrg)
+          ? testContext.createError({
+            message: `You must provide the ${Settings.fields.eventSeries.hostOrg.label}`
+          })
+          : true
+      )
+      .default({}),
+    adminOrg: yup
+      .object()
+      .test("adminOrg", "admin org error", (adminOrg, testContext) =>
```

Comment:
Can the event administrator organization be made optional?

Reply:
Is optional now.